docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst

```diff
@@ -25,3 +27,104 @@ for GBD are EHB from `Rhesus (Rh) disease`_, preterm birth,
 
 .. todo::
    Describe cause model
+
+Data Descriptions
+-----------------
+
+Preliminary!
+
+.. list-table:: Definitions
+	:widths: 5 10 10
+	:header-rows: 1
+	
+	* - State
+	  - State name
+	  - Definition
+	* - C
+	  - With **C**\ ondition
+	  - Simulant was born with haemolytic disease or other neonatal jaundice
+	* - F
+	  - **F**\ ree of condition
+	  - Simulant was not born with haemolytic disease or other neonatal jaundice
+	  
+.. list-table:: States
+	:widths: 5 10 10 20
+	:header-rows: 1
+	
+	* - State
+	  - Measure
+	  - Value
+	  - Notes
+	* - C
+	  - prevalence
+	  - prev_c384
+	  -
+	* - C
+	  - birth prevalence
+	  - birth_prev_unknown
+	  - is this the table we want to house this in?
+	* - C
+	  - emr
+	  - :math:`\frac{\text{deaths_c384}}{\text{population} \,\times\, \text{prevalence_c384}}`
+	  -
+	* - C
+	  - disability weight
+	  - :math:`\displaystyle{\sum_{s\in \text{sequelae_c384}}} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}`
+	  -
+	* - F
+	  - prevalence
+	  - 1-prev_c384
+	  -
+	* - F
+	  - birth prevalence
+	  - 1-birth_prev_unknown
+	  - is this the table we want this in?
+	* - F
+	  - emr
+	  - 0
+	  -
+	* - F
+	  - disability weight
+	  - 0
+	  -
+	* - All
+	  - CSMR
+	  - :math:`\frac{\text{deaths_c384}}{\text{population}}`
+	  -
+	 
+	  
+.. list-table:: Transitions
+	:widths: 10 10 10 10 10
+	:header-rows: 1
+	
+	* - Transition
+	  - Source State
+	  - Sink State
+	  - Value
+	  - Notes
+	* - N/A
+	  - N/A
+	  - N/A
+	  - N/A
+	  - N/A
+	  
+.. list-table:: Data Definitions
+	:widths: 10 10 20
+	:header-rows: 1
+	
+	* - Variable
+	  - Source
+	  - Notes
+	* - prev_c384
+	  - como
+	  - 
+	* - birth_prev_unknown
```

Comment:
If there is a value we can pull directly, we should call it birth_prevalence_c384, and explain how to get it in the notes column. If we have to combine multiple raw values to get the right number, we need to list each of those in this table.